python/lammps.py

```diff
@@ -506,6 +506,8 @@ def __init__(self,name='',cmdargs=None,ptr=None,comm=None):
     self.lib.lammps_set_fix_external_callback.argtypes = [c_void_p, c_char_p, self.FIX_EXTERNAL_CALLBACK_FUNC, py_object]
     self.lib.lammps_set_fix_external_callback.restype = None
 
+    self.mliappy = MLIAPPY(self)
```

Comment:
@nicklubbers says he posted a response to above comments earlier today, but it is awaiting moderator approval.  Can he be added to a  pre-approved list?

Reply:
> @lubbersnick says he posted a response to above comments earlier today, but it is awaiting moderator approval. Can he be added to a pre-approved list?

I am not aware of having any moderation feature enabled for lammps on GitHub.

Reply:
Fixed now. It didn't say anything about "moderators", just listed "pending" next to the comments. I guess it had something to do with my leaving a comment on a part of the code nobody had discussed yet. Sorry about that.

Reply:
> Fixed now. It didn't say anything about "moderators", just listed "pending" next to the comments. I guess it had something to do with my leaving a comment on a part of the code nobody had discussed yet. Sorry about that.

The "pending" status happens when you click on "Start a review" while the behavior you expected would happen with clicking on the "Add single comment" button.

Reply:
@lubbersnick I agree that `lammps` should be a package. I had that on my agenda for some time now, but it was always pushed back for some reason (e.g., stable release). I'll use this as an excuse to start that process. I'll create a new PR for this change.

Reply:
I am happy to try and help refactoring the file if there is a clear method for me to test my changes, feel free to involve me about that. The part where I'm still pretty new is on the build/configuration side.

For this PR are we ok to add `class MLIAPPY()` to the end of lammps.py? Would you prefer if we moved mliappy_pytorch.py to a new directory /python/mliappy/ for the moment, and update the example so instead of copying, it add this to the sys.path?

Reply:
@lubbersnick I started with PR #2521. Let me know what you think. For this PR I'm ok having it in lammps.py for now. I only would ask to only create a MLIAPPY instance when accessed, similar to what you see with the optional numpy wrapper. Having that folder and adding it to the sys path sound good for now. It would probably end up in a subpackage, like `python/lammps/mliappy/`